dev/com.ibm.ws.artifact.file/src/com/ibm/ws/artifact/file/FileArtifactNotifier.java

```diff
@@ -291,6 +291,8 @@ private synchronized void updateFileMonitorService() {
                 return;
             }
             serviceProperties.put(Constants.SERVICE_VENDOR, "IBM");
+            // Adding the com.ibm.ws.kernel.monitor.app parameter to the monitors that gets registered to use in new mbean update command
```

Comment:
I don't think FileArtifactNotifier is the best place to put these changes. The container services code is SPI, so it's possible someone may set up a container and not expect it to follow the same update cycle as an application.